File: app/src/main/java/com/topnotch/developers/model/GithubUsersResponse.java
package com.topnotch.developers.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GithubUsersResponse {

    @SerializedName("total_count")
    public Integer totalCount;
    @SerializedName("incomplete_results")
    public Boolean incompleteResults;
    @SerializedName("items")
    public List<GithubUser> githubUsers = null;

    public Integer getTotalCount() {
        return totalCount;
    }

    public Boolean getIncompleteResults() {
        return incompleteResults;
    }

    public List<GithubUser> getGithubUsers() {
        return githubUsers;
    }
}

File: app/src/main/java/com/topnotch/developers/model/GithubUsersResponse.kt
package com.topnotch.developers.model

import com.google.gson.annotations.SerializedName

data class GithubUsersResponse(
    @SerializedName("total_count") val totalCount: Int = 0,
    @SerializedName("incomplete_results") val incompleteResults: Boolean = false,
    @SerializedName("items") val items: List<GithubUser>? = null
)